style.css

```diff
@@ -342,3 +360,174 @@ button:hover {
   color: white;
   background: #396df2;
 }
+
+/* ---------------------------------------------Desktop version--------------------- */
+@media screen and (max-width: 768px) {
+  .menu {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .bar-button {
+    display: none;
+  }
+
+  .menu {
+    display: flex;
+  }
+
+  .navbar {
+    margin-left: 155px;
+    margin-right: 155px;
+  }
+
+  .header {
+    padding-bottom: 130px;
+  }
+
+  .menu a {
+    margin-right: 32px;
+    font-weight: 500;
+    font-size: 15px;
+    color: #344563;
+  }
+
+  .background {
+    background-image: url(./images/Header\ bg@2x.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
```

Comment:
- Good job using media queries properly 👏 . But there seem to be a few differences in this section of your design from what is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- The background image for this section is not properly aligned, please reference the marked images in the design template I attached below.
<img width="948" alt="werewv" src="https://user-images.githubusercontent.com/73607512/206393824-bc78d3d0-757d-4cb4-854d-34275cc40916.PNG">

Figma Design:
<img width="569" alt="rvevev" src="https://user-images.githubusercontent.com/73607512/206394125-e492c851-3509-469f-ab6f-536e4ab06ed7.PNG">